### DISCUSSION TITLE
Use `mathml` in `hugo-md` format

### DISCUSSION THREAD

**Discussion by AlbertLei** [quarto-dev/quarto-cli#5527] [Q&A] [answered]
**Background.** I am using quarto to render some math-intensive pages in a hugo project. I prefer `mathml` for math rendering. It's much faster than `webtex` (which is recommended in the quarto webpage for `hugo-md` format). The math rendering of `mathml` also seems much nicer to me (I am using firefox). 

I tried specifying `html-math-method: mathml` in the `hugo-md` format. But the generated md file still keeps the dollar-latex-dollar syntax, rather than mathml syntax. I am using quarto 1.2.335.   

So, my question is, is it possible to support `html-math-method: mathml` in the `hugo-md` format? My (naive) guess is that may not be hard, since one can mix mathml (which is just HTML) and markdown in a hugo-legitimate md file. 

Currently I am using the workaround by specifying `format: html`, `theme: none` and `html-math-method: mathml`. It works very good for me as (i) hugo allows raw html file and (ii) mathml is valid in the html output of quarto.

**Comment by cderv** [comment, marked as answer]
`mathml` is supported only [for HTML outputs by Pandoc](https://pandoc.org/MANUAL.html#math-rendering-in-html) and `hugo-md` is based on a Markdown output. This means specifying `mathml` as a method won't do anything for markdown output. 

This seems to be a feature request for Pandoc to support outputing some mathml math as HTML in some markdown output supporting HTML. 

I am not sure we have an easy way to do this in place of Pandoc's internal for math support.

**Reply by AlbertLei** [reply]
Thanks for letting me know the limitation of pandoc.
I am fine with the `format: html` workaround.
